### DISCUSSION TITLE
Geophysical data in the Arctic

### DISCUSSION THREAD

**Discussion by Bohr-niels** [mdtanker/antarctic_plots#206] [Dataset requests]
I think adding some  Arctic geophysical data could broaden the application fields of the toolkit, such as the bathymetric datasets "IBCAO V4", which can be downloaded from the url: https://www.gebco.net/data_and_products/gridded_bathymetry_data/arctic_ocean/

**Comment by mdtanker** [comment]
Thanks again for the suggestions. I've just released `v0.4.0` which now contains a number of Arctic / Greenland datasets which can be seen [here](https://polartoolkit.readthedocs.io/en/latest/datasets/greenland_arctic/index.html). 

Currently these include:
* bedmachine ice surface, ice base and bed topography, and ice thickness
* geoid anomaly
* surface topography from ETOPO1
* MODIS Mosaic of Greenland Imagery
* Ice mass change
* Ice velocity
* Greenland grounding line shapefiles

I'm happy to try and add IBCAO at some point if you'd find it useful!
